### ISSUE TITLE
Loss doesn't decrease throughout the entire training 

### ISSUE THREAD

**Issue by aswahd** [weih527/Pixel-Embedded-Affinity#1]
Thank you for open sourcing your work.

I am training on CVPPP dataset. I used the default training config, however, the loss doesn't decrease and the gradients are very large.

Example:
```
model.down1.block.conv[3].weight.grad.max()
>>
tensor(878.7301, device='cuda:0')
```

The log file for loss is:
```
step = 197900, loss = 4364.775894, loss_emd = 2629.880396, loss_mask = 0.000000
step = 198000, loss = 4395.702788, loss_emd = 2651.596763, loss_mask = 0.000000
step = 198100, loss = 4406.793435, loss_emd = 2657.019900, loss_mask = 0.000000
step = 198200, loss = 4388.137615, loss_emd = 2643.711174, loss_mask = 0.000000
step = 198300, loss = 4318.416519, loss_emd = 2603.078289, loss_mask = 0.000000
step = 198400, loss = 4368.949673, loss_emd = 2636.413052, loss_mask = 0.000000
step = 198500, loss = 4285.260076, loss_emd = 2583.959093, loss_mask = 0.000000
step = 198600, loss = 4380.306262, loss_emd = 2645.390134, loss_mask = 0.000000
step = 198700, loss = 4551.785659, loss_emd = 2758.221311, loss_mask = 0.000000
step = 198800, loss = 4425.482607, loss_emd = 2673.479055, loss_mask = 0.000000
step = 198900, loss = 4331.416313, loss_emd = 2612.468118, loss_mask = 0.000000
step = 199000, loss = 4476.659573, loss_emd = 2701.468428, loss_mask = 0.000000
step = 199100, loss = 4396.204197, loss_emd = 2651.603896, loss_mask = 0.000000
step = 199200, loss = 4377.829529, loss_emd = 2635.787970, loss_mask = 0.000000
step = 199300, loss = 4381.929780, loss_emd = 2643.542228, loss_mask = 0.000000
step = 199400, loss = 4519.812466, loss_emd = 2736.273082, loss_mask = 0.000000
step = 199500, loss = 4372.386707, loss_emd = 2641.280640, loss_mask = 0.000000
step = 199600, loss = 4415.834993, loss_emd = 2664.761194, loss_mask = 0.000000
step = 199700, loss = 4379.413796, loss_emd = 2644.091644, loss_mask = 0.000000
step = 199800, loss = 4312.231597, loss_emd = 2602.108986, loss_mask = 0.000000
step = 199900, loss = 4454.877102, loss_emd = 2685.370757, loss_mask = 0.000000
step = 200000, loss = 4300.497131, loss_emd = 2593.497546, loss_mask = 0.000000
```

The loss stays at roughly the same value.

What could possibly be the reason for the high gradient and non-decreasing loss.

Thank you.
